test/test-dtutils.lua

```diff
@@ -0,0 +1,116 @@
+--[[
+
+    test-dtutils.lua - the start of a testing framework idea....
+
+    copyright (c) 2016 Bill Ferguson
+
+    TODO: Think this out a little better
```

Comment:
I was working on this last night and got disgusted with it.  Today I was looking at luaunit.  I'm a little concerned about introducing external dependencies.  I can get luaunit using luarocks, or I can download it and stick it somewhere in the tree.  bin/luaunit perhaps?  I also looked at busted, but I think it's a little too much for this.